src/kirby/components/icon/icon.component.tns.global.scss

```diff
@@ -2,32 +2,37 @@
 
 .kirby-icon {
     display: none;
+    //default to icon size 'sm'
     Label {
         font-family: "Ionicons", "ionicons";
         text-align: center;
+        font-size: size('m');
 
         &.outline {
+            width: #{size('m') + (size('xs') * 2)};
+            height: #{size('m') + (size('xs') * 2)};
```

Comment:
??

Reply:
If width / height aren't set on native, I experienced that the label messed up the context (native). This is why I set the width which is using the size of the icon + "padding" to the border